vendor/github.com/v3io/v3io-go-http/synccontainer.go

```diff
@@ -236,6 +236,15 @@ func (sc *SyncContainer) GetItems(input *GetItemsInput) (*Response, error) {
 		return nil, err
 	}
 
+	//validate range scan loop
```

Comment:
Open a PR to v3io-go-http and then we will re-vendor with the latest v3io-go-http version.

Reply:
already did - this is the only change

Reply:
This diff doesn't match https://github.com/v3io/v3io-go-http/pull/14